app/lib/language-engine.js

```diff
@@ -0,0 +1,24 @@
+import counterpart from 'counterpart';
+
+const load = (language) => {
+  const defaultLanguage = { label: 'English', value: 'en' };
```

Comment:
Should this default to the project primary language if the project isn't in English?